queue_job/migrations/10.0.1.0.0/pre-migration.py

```diff
@@ -25,5 +28,12 @@ def migrate(cr, version):
                  'queue_job.group_queue_job_manager',)
             ],
         )
+        # Pre-create column for avoiding the computation over non loaded models
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        openupgrade.add_fields(env, [
+            ('channel_method_name', 'queue.job', 'queue_job', 'char', False,
+             'queue_job'),
+        ])
     except ImportError:
-        pass
+        # Create at least the column, although without XML-ID
```

Comment:
@pedrobaeza Why do you mute this ImportError? IMO openupgradelib should be mandatory if the migration script runs.

Reply:
@lmignon yeah I prefer that way, but others like @hbrunn or @jbeficent  have told me in other occasions that they don't want to have it as mandatory

Reply:
that must have been a misunderstanding. I think it's fine to mandate openupgradelib, but then it must be a python dependency of the module and a requirement

Reply:
Yeah, but I didn't want to put that dependency in the manifest as it's not required for normal module installation, but only on upgrades, so we are in a circular loop...

Reply:
ah, that was the point then. Yes, I firmly believe if it's not a requirement, it can't be required for anything. Positive review added ;-)

Reply:
I've no problem to not declare the dependency into the manifest since it's only used in some specific UC (migration from 9. to 10) but if it is not present in such circumstances the error must be raised.

Reply:
OK, @lmignon, I have raise an exception if `openupgradelib` is not found.